universal-login-react/src/ui/Onboarding/EmailFlowChooser.tsx

```diff
@@ -27,39 +29,54 @@ export const EmailFlowChooser = ({onCreateClick, onConnectClick}: EmailFlowChoos
         <div className={useClassFor('user-tabs')}>
           <button
             onClick={() => setFlow({kind: 'create', onClick: onCreateClick})}
-            className={`${classForComponent('user-tab')} ${flow.kind === 'create' ? 'active' : ''}`}>New user</button>
+            className={` ${classForComponent('user-tab')} ${flow.kind === 'create' ? 'active' : ''}`}>New user</button>
           <button
             onClick={() => setFlow({kind: 'connect', onClick: onConnectClick})}
-            className={`${classForComponent('user-tab')} ${flow.kind === 'connect' ? 'active' : ''}`}>Existing user</button>
+            className={` ${classForComponent('user-tab')} ${flow.kind === 'connect' ? 'active' : ''}`}>Existing user</button>
         </div>
         <div className={useClassFor('flow-content')}>
-          {flow.kind === 'create' && <CreationContent/>}
-          {flow.kind === 'connect' && <ConnectionContent/>}
+          {flow.kind === 'create' && <CreationContent email={email} setEmail={setEmail} ensName={ensName} setEnsName={setEnsName} />}
+          {flow.kind === 'connect' && <ConnectionContent />}
```

Comment:
shouldn't we pass here ensName or e-mail?

Reply:
I guess it should be more ensNameOrEmail (regarding to designs). We can introduce also a helper function that will tell us what type this string is.